code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -22,16 +22,14 @@
 	return ..()
 
 /obj/item/pneumatic_cannon/examine(mob/user)
-	..()
+	. = ..()
 	if(!in_range(user, src))
-		to_chat(user, "<span class='notice'>You'll need to get closer to see any more.</span>")
-		return
-	for(var/obj/item/I in loadedItems)
-		spawn(0)
```

Comment:
This spawn was probably here to show the items on the bottom of the text. No clue why it was done like this but hey